src/main/java/com/gary/backendv2/controller/FacilityController.java

```diff
@@ -0,0 +1,44 @@
+package com.gary.backendv2.controller;
+
+import com.gary.backendv2.model.dto.request.FacilityRequest;
+import com.gary.backendv2.service.FacilityService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/facility")
+public class FacilityController {
+	private final FacilityService facilityService;
+
+	@GetMapping
+	public ResponseEntity<?> getAll(){
+		return ResponseEntity.ok(facilityService.getAll());
```

Comment:
Please create response dto to make frontend interaction easier. Don't return entire object from the model.